README.md

```diff
@@ -25,7 +25,7 @@ For user friendly details about Pleroma: [see here](https://blog.soykaf.com/post
 - Tootdon (Android + iOS)
 - Tootle (iOS)
 
-**Shipped version:** 0.9.99999
+**Shipped version:** 1.0.1
```

Comment:
Maybe you can also update the README to say that LDAP is now supported. :smile:

Reply:
It's planned to update the readme, just waiting to have ldap implemented in the package ^^